timemachine/fe/atom_mapping.py

```diff
@@ -411,6 +447,27 @@ def planar_torsion_flip_filter(trial_core):
     def filter_fxn(trial_core):
         return all(f(trial_core) for f in filter_fxns)
 
+    def leaf_filter_fxn(g1: nx.Graph, g2: nx.Graph, atom_map_1_to_2, atom_map_2_to_1) -> bool:
+        if disallow_chiral_conversion:
+            g1_mapped_nodes = {a1 for a1, a2 in enumerate(atom_map_1_to_2) if a2 != UNMAPPED}
+            g1_core_disabled_bonds = compute_disabled_bonds_in_core(g1, g2, g1_mapped_nodes, atom_map_1_to_2)
+            if enforce_core_core:
+                assert len(g1_core_disabled_bonds) == 0
+
+            if _graph_fails_chiral_assertion(g1, g1_mapped_nodes, g1_core_disabled_bonds):
+                return False
+
+            g2_mapped_nodes = {a2 for a2, a1 in enumerate(atom_map_2_to_1) if a1 != UNMAPPED}
+            g2_core_disabled_bonds = compute_disabled_bonds_in_core(g2, g1, g2_mapped_nodes, atom_map_2_to_1)
+
+            if enforce_core_core:
+                assert len(g2_core_disabled_bonds) == 0
```

Comment:
integration q: should this line raise an assertion or return a boolean?

Reply:
Shouldn't be relevant anymore since https://github.com/proteneer/timemachine/pull/1356/commits/546037d42cb12ff9af3303002f77d09c122c2ef9